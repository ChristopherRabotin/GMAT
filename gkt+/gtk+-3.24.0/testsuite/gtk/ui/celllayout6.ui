<!-- test invalid cell layout custom attribute value -->
<interface>
  <object class="GtkIconView">
    <child>
      <object class="GtkCellRendererText">
      </object>
      <cell-packing>
        <property name="name" food="food">1</property>
      </cell-packing>
    </child>
  </object>
</interface>
