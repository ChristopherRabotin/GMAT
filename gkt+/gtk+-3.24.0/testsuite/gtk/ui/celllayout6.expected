ERROR: g-markup-error-quark 4
.:8:44 attribute 'food' invalid for element 'property'
